packages/react/src/elements/components/GlobalNav/TopNav/Notifications.js

```diff
@@ -11,20 +11,46 @@ class Notifications extends Component {
     };
   }
 
+  componentDidMount = () => {
+    const showNotificationsCount = this.props.children.length > 0;
```

Comment:
Let's bring back those checks for the existence of `this.props.children`, or else this will complain of calling length of `undefined`.